client/state/themes/actions.js

```diff
@@ -765,3 +768,42 @@ export function confirmDelete( themeId, siteId ) {
 		);
 	};
 }
+
+function setPreviewingThemeData( themeId, source ) {
```

Comment:
Final nomenclature nit -- naming should be consistent with the function below. What about `setThemePreviewData` and `setThemePreviewOptions`?